### DISCUSSION TITLE
Cant Access Security Onion '2.3.40 Dashboard (GUI Interface) on host machine from virtual box

### DISCUSSION THREAD

**Discussion by mbunde** [Security-Onion-Solutions/securityonion#3880] [Unsupported Versions]
Hi team,

I have been facing challenges accessing security onion interface from my host machine. Security Onion is installed on Virtual-box.
I have also done so-allow to enable access. I can also ping the IP from my host machine but when  trying opening from the   browser it refuses. I have disabled my windows firewall but still no change.

Kindly asking for your advice.

**Comment by pboosten** [comment]
Hi,

Virtualbox is quite finicky with network settings. What did you set the interfaces to? I'm using 'bridged' mode, and that works quite well, but you need an IP address in your current network.

Look [here](https://www.virtualbox.org/manual/ch06.html) for options.

Peter

**Comment by mbunde** [comment]
Hi  Thanks a lot for the info

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by mbunde** [comment]
Hi Pboosten,

Thanks for the feedback. I resorted to use bridged.
Now I got  different challenge, when I try to access the SOC interface on the host machine I get  error permission denied
***@***.***D73457.97AE2350]
However, I noted that I might have configured the SOC dashboard access to  the IP matching my host machine. I supplied I CDR  like this 192.168.100 .2/24. The IP 192.168.100 .2 is my host machine IP that has been configured for the SOC dashboards access as well.

Could this be the reason I am getting the error. Incase I update the IP  to 192.168.100 .3 would it solve the prblm ?

Looking forward to hearing from you

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by pboosten** [reply]
Hi,

You probably misunderstood the info from Virtualbox: when using 'bridged' mode, the virtual machine (running in virtualbox) gets its own MAC address, and needs its own IP address. That probably means that you need to change the SO configuration as well. In bridged mode, your host is a different machine than your virtual machine.

What were you coming from? If you used NAT before, then your virtual machine gets to use the host IP address, but you need to configure port forwarding to have your host machine forward the traffic for a certain port to the port on your virtual machine. Port forwarding is configured in virtualbox (within the network settings -> this is greyed out when not using NAT mode).

Peter

**Comment by mbunde** [comment]
Hi Thanks Peter,

I will reconfigure it.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
